src/openssl_wrap.cpp

```diff
@@ -1323,5 +1323,29 @@ std::vector<uint8_t> _EC_KEY_key2buf(const EVP_PKEY* evp, point_conversion_form_
     return result;
 }
 
+
+std::vector<uint8_t> _ECDH_compute_key(const EVP_PKEY *pub_key, const EVP_PKEY *ecdh,
+                                       void *(*KDF) (const void *in, size_t inlen, void *out, size_t *outlen))
+{
+    EVP_PKEY *pub_key_ = const_cast<EVP_PKEY*>(pub_key);
+    EVP_PKEY *ecdh_ = const_cast<EVP_PKEY*>(ecdh);
+    EC_KEY *ec_key = OpensslCallPtr::callChecked(lib::OpenSSLLib::SSL_EVP_PKEY_get1_EC_KEY, ecdh_);
+    const EC_GROUP *ec_group = OpensslCallPtr::callChecked(lib::OpenSSLLib::SSL_EC_KEY_get0_group, ec_key);
+    EC_KEY *pub_ec_key = OpensslCallPtr::callChecked(lib::OpenSSLLib::SSL_EVP_PKEY_get1_EC_KEY, pub_key_);
```

Comment:
Same issue here, the `EVP_PKEY_get1_EC_KEY` function increases the reference count and should come with an associated free.

Reply:
Done